vaadin-testbench-unit/src/main/java/com/vaadin/testbench/unit/UIUnit4Test.java

```diff
@@ -23,19 +31,46 @@
  * {@code @Before} methods defined in subclasses. At the same way, cleanup tasks
  * operated by {@link #cleanVaadinEnvironment()} are executed after each test,
  * and after all {@code @After} annotated methods in subclasses.
- *
  */
```

Comment:
Can we add here the paragraph with info on how to setup pretty print, as we have in UIUnitTest?